opentreemap/treemap/js/src/lib/inlineEditForm.js

```diff
@@ -19,6 +19,7 @@ exports.init = function(options) {
         form = options.form || section + ' form',
         $section = $(section),
         edit = options.edit || '.editBtn',
+        $edit = $(edit),
```

Comment:
We always include our JS at the bottom of the `<body>`, so assuming the markup is loaded before scripts are parsed and run is safe.  https://github.com/OpenTreeMap/otm-core/blob/bec99c0e93495b6e65f0bc293249fd37dd045fa4/opentreemap/treemap/templates/base.html#L179-L217